src/flows/components/header/PublishedVersions.tsx

```diff
@@ -0,0 +1,30 @@
+import React, {FC, useContext, useEffect} from 'react'
+import VersionBullet from 'src/flows/components/header/VersionBullet'
+import {VersionPublishContext} from 'src/flows/context/version.publish'
+import {isFlagEnabled} from 'src/shared/utils/featureFlag'
+
+const PublishedVersions: FC = () => {
+  const {versions, handleGetNotebookVersions} = useContext(
+    VersionPublishContext
+  )
+
+  useEffect(() => {
```

Comment:
oh make the provider responsible for fetching

Reply:
K